### DISCUSSION TITLE
[i3blocks] Show custom tooltip on hover?

### DISCUSSION THREAD

**Discussion by CarterSnich** [i3/i3#6133] [Q&A] [answered]
Is it possible to show a tooltip when hovering a block? I want to show the complete date when hovering my clock block.

**Comment by okraits** [comment, marked as answer]
I don't think it is possible. Please read the documentation of i3blocks: https://vivien.github.io/i3blocks/ Please also note that i3blocks is not a project maintained by the developers of i3.
